src/pe-csm-project-details/widget/pe-csm-project-details.u-client-script.js

```diff
@@ -0,0 +1,73 @@
+function($scope, $http, $window) {
+  /* widget controller */
```

Comment:
With a named controller function in place; which is self descriptive of purpose, you can feel free to remove the /* widget controller */ comment.